test/bdd/pkg/introduce/introduce_sdk_steps.go

```diff
@@ -396,3 +407,43 @@ func (a *SDKSteps) sendRequest(introducee1, introducer, introducee2 string) erro
 
 	return a.clients[introducee1].SendRequest(to, conn1.MyDID, conn1.TheirDID)
 }
+
+func (a *SDKSteps) newOOBRequest(agentID string) (*outofband.Request, error) {
+	client, err := outofband.New(a.bddContext.AgentCtx[agentID])
+	if err != nil {
+		return nil, err
+	}
+
+	r, err := client.CreateRequest(
+		outofband.WithLabel(agentID),
+		outofband.WithAttachments(&decorator.Attachment{
+			ID:          uuid.New().String(),
+			Description: "test",
+			Data: decorator.AttachmentData{
+				JSON: map[string]interface{}{},
+			},
+		},
+		))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit: redundant condition

Reply:
I get you, but I've had experience with people in the past wanting to ensure that the result value is `nil` if a function is returning an error.